xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp

```diff
@@ -62,6 +62,9 @@ using namespace PVR;
 #define SETTING_TMR_DIR           "timer.directory"
 #define SETTING_TMR_REC_GROUP     "timer.recgroup"
 
+#define MARGIN_SPINNER_STEP 15
```

Comment:
If you have a step and a maximum defined in settings.xml, why repeat it here as a #define?
This looks like confusion waiting to happen: which setting will control kodi's behaviour?
Surely the value should be read from settings.xml?